website/content/docs/enterprise/license.mdx

```diff
@@ -20,24 +20,19 @@ source version of Nomad. Servers running the open source version of Nomad will
 panic if they are joined to a Nomad Enterprise cluster. See issue [gh-9958]
 for more details.
 
-## Evaluating Nomad Enterprise
-
-Nomad Enterprise can be used without a valid license for 6 hours. When a Nomad
-Enterprise server starts without a license configuration option (see [license
-configuration]) it uses a temporary trial license. This license is valid
-for 6 hours.
+## Expiring Licenses
 
-You can inspect the temporary license using `nomad license get`.
+Nomad Enterprise license have an expiration time. You can read the license on
+a server with the `nomad license get` command:
 
 ```
 $ nomad license get
 Product        = nomad
 License Status = valid
-License ID     = temporary-license
-Customer ID    = temporary license customer
+License ID     = my-license
+Customer ID    = my license customer
```

Comment:
With temporary licenses gone these should only ever be UUIDs, with the exception of on prem, but I think uuids here might be more clear.

```suggestion
License ID     = eaaecbd5-b598-4fa4-b649-bb234bf49fa3
Customer ID    = ae94bdd0-39bf-4c58-bf80-8a9468b0467c
```